helpers/kernel_config.go

```diff
@@ -87,9 +87,10 @@ const (
 	CONFIG_BPF_LSM
 	CONFIG_BPF_PRELOAD
 	CONFIG_BPF_PRELOAD_UMD
+	CUSTOM_OPTION_MARK KernelConfigOption = 1000
```

Comment:
not a must, but maybe CUSTOM_OPTION_START would be a better fit?

Reply:
I currently don't have a way to push without a review. I'm changing it and will kindly ask you @yanivagman to approve this again (BTW, should we change this policy for nit fixes ? Do we always want to enforce nit fixes push with a big turn around of 1 or 2 days, until the review is done again ?)